### ISSUE TITLE
var foo = Math.PI * arr[0]; Error CS0019: Operator `*' cannot be applied to operands of type `double' and `dynamic' (CS0019)

### ISSUE THREAD

**Issue by sushihangover** [PlayScriptRedux/playscript#59]
Regression from away3d-core-fp11:

```
tmpVecs[j].x = Math.cos(-angle/180*Math.PI) *  (aRads[j] + tweekradius );
```

Produces:

```
/Users/administrator/Documents/Code/playscript/playscriptredux/away3d-core-fp11/src/away3d/extrusions/LatheExtrude.as(28,28): Error CS0019: Operator `*' cannot be applied to operands of type `double' and `dynamic' (CS0019) (away3d-core-monomac)
```

**Comment by sushihangover** [comment]
#### New test:

as/test-as-DoubleTimesDynamic-CS0019.as

```
// Compiler options: -psstrict-
package {
    public class Foo {
        public static function Main():int {
                var arr:Array = [1.0, 1.5, 2.0];
                trace(arr[0]);
                trace(Math.PI);

                // Should not produce:

                // error CS0019: Operator `*' cannot be applied to
                // operands of type `double' and `dynamic'

                var foo = Math.PI * arr[0];

                return 0;
        }
    }
}
```

**Comment by sushihangover** [comment]
commit 16f9c8b98b022708e1a515893ef7d9ed0665a6c1
Author: SushiHangover sushihangover@outlook.com
Date:   Fri Aug 14 23:32:52 2015 -0700

```
Fixes #59 : Operator `*' cannot be applied to operands of type XXXX and `dynamic'
```
